addons/payment_stripe/models/payment_transaction.py

```diff
@@ -54,15 +54,24 @@ def _stripe_create_checkout_session(self):
         :return: The Checkout Session
         :rtype: dict
         """
+        def get_linked_pmts():
+            linked_pmts_ = linked_pms
```

Comment:
confusing to reference `linked_pms` here without giving it as param.

Reply:
I don't, because it's expected from inner functions to have access to their outer function's namespace. I added the param, nevertheless.

Reply:
I wasn't doubting it was working, just finding it confusing :D